src/functions.php

```diff
@@ -46,6 +47,26 @@
 use function MongoDB\BSON\toPHP;
 use function substr;
 
+/**
+ * Registers a PSR-3 logger to receive log messages from the driver/library.
+ *
+ * Adding a registered logger is a NOP.
```

Comment:
"NOP" is a jargon I don't use.

```suggestion
 * Calling this method again with a logger that has already been added will have no effect.
```